index.html

```diff
@@ -31,6 +31,7 @@
         </main>
     </div>
     <main id="portfolio">
+        <h4 class="title brand">My Projects</h4>
```

Comment:
Do you need this 'title' class at all? Couldn't you just do this on the h4 and then change the colour using .brand as needed?